hw_16/src/calculator.ts

```diff
@@ -0,0 +1,31 @@
+export function calculator(
+  firstNumber: number,
+  secondNumber: number,
+  operator: '+' | '-' | '/' | '*'
+) {
+  switch (operator) {
+    case '+':
+      if (operator === '+') {
+        return firstNumber + secondNumber;
+      }
```

Comment:
Why do you need `if` statement, if you already have `switch..case` construction?